poi/src/test/java/org/apache/poi/ss/formula/functions/TestDays.java

```diff
@@ -37,6 +39,11 @@ Licensed to the Apache Software Foundation (ASF) under one or more
  */
 public class TestDays {
 
+    @BeforeAll
+    static void setup() {
+        Locale.setDefault(Locale.US);
```

Comment:
POI has its own LocaleUtil. Could you use that instead?

Reply:
Sure, done